tests_e2e/tests/ext_sequencing/ext_sequencing.py

```diff
@@ -61,6 +61,8 @@ class ExtSequencing(AgentVmssTest):
         add_failing_dependent_extension_with_two_dependencies
     ]
 
+    _scenario_start = datetime.min
```

Comment:
this should be an instance member rather than a class member, since multiple instances of the tests may be executing concurrently